apis/sgv2-restapi/src/test/java/io/stargate/sgv2/it/RestApiV2QIntegrationTestBase.java

```diff
@@ -681,4 +767,131 @@ protected void setupMixedClusteringTestCase(String keyspaceName, String tableNam
     row.put("v", 18);
     insertRow(keyspaceName, tableName, row);
   }
+
+  protected String updateRowReturnResponse(
+      String updatePath, boolean raw, Map<?, ?> payload, Boolean compactMapData) {
+    return updateRowReturnResponse(updatePath, raw, payload, HttpStatus.SC_OK, compactMapData);
+  }
+
+  protected String updateRowReturnResponse(String updatePath, boolean raw, Map<?, ?> payload) {
+    return updateRowReturnResponse(updatePath, raw, payload, HttpStatus.SC_OK);
+  }
+
+  protected String updateRowReturnResponse(
+      String updatePath,
+      boolean raw,
+      Map<?, ?> payloadMap,
+      int expectedStatus,
+      Boolean compactMapData) {
```

Comment:
Boolean -> boolean?

Reply:
This needs to be Boolean. There's a null check down stream.